src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/DeriveHearingCentreHandler.java

```diff
@@ -117,19 +117,15 @@ private Optional<String> getAppealPostcode(
             }
         }
 
-        if (asylumCase.read(APPELLANT_HAS_FIXED_ADDRESS, YesOrNo.class)
-                .orElse(NO) == YES) {
-
-            Optional<AddressUk> optionalAppellantAddress = asylumCase.read(APPELLANT_ADDRESS);
+        Optional<AddressUk> optionalAppellantAddress = asylumCase.read(APPELLANT_ADDRESS);
 
-            if (optionalAppellantAddress.isPresent()) {
-                AddressUk appellantAddress = optionalAppellantAddress.get();
-                return appellantAddress.getPostCode();
-            }
+        if (optionalAppellantAddress.isPresent()) {
+            AddressUk appellantAddress = optionalAppellantAddress.get();
+            return appellantAddress.getPostCode();
         }
 
         Optional<AddressUk> optionalLegalRepCompanyAddress =
-                asylumCase.read(LEGAL_REP_COMPANY_ADDRESS, AddressUk.class);
```

Comment:
Some scenarios for non-EJP legally represented appeals will still need to use this field to determine the hearing centre so we will need to maintain this logic